components/layout/lib.rs

```diff
@@ -8,8 +8,11 @@
 #![feature(custom_derive)]
 #![feature(nonzero)]
 #![feature(plugin)]
+#![feature(proc_macro)]
 #![feature(raw)]
 #![feature(step_by)]
+#![feature(structural_match)]
+#![feature(rustc_attrs)]
```

Comment:
Oh we don't need that anymore, I can compile with `rustc-1.14.0` with them removed.